pkg/config/config_test.go

```diff
@@ -285,6 +286,29 @@ func TestComposeConfig(t *testing.T) {
 			Platform:         configv1.OpenStackPlatformType,
 			IsSingleReplica:  true,
 		},
+	}, {
+		name:      "With Cluster Wide Proxy",
+		namespace: defaultManagementNamespace,
+		platform:  configv1.AWSPlatformType,
+		clusterProxy: &configv1.Proxy{
+			Status: configv1.ProxyStatus{
+				HTTPProxy: "http://squid.corp.acme.com:3128",
+			},
+		},
+		imagesContent: `{
+    "cloudControllerManagerAWS": "registry.ci.openshift.org/openshift:aws-cloud-controller-manager",
+    "cloudControllerManagerOpenStack": "registry.ci.openshift.org/openshift:openstack-cloud-controller-manager"
+}`,
```

Comment:
seems like the indent is a little off here?

Reply:
It's in all places. https://github.com/openshift/cluster-cloud-controller-manager-operator/blob/master/pkg/config/config_test.go#L37-L58
we can fix it in a separate commit.